.github/workflows/run-addon.yml

```diff
@@ -4,4 +4,43 @@ on: repository_dispatch
 
 jobs:
   Run-Add-On:
-    uses: MuckRock/documentcloud-addon-workflows/.github/workflows/run-addon.yml@v1
+    runs-on: ubuntu-latest
```

Comment:
Is there some way to use the referenced workflow and add on the NLTK stuff?  That way you get any updates that are made to the standard workflow

Reply:
The problem with the referenced workflow is that it runs `python main.py`, and the NTLK stuff must be there before `main.py` is run, so I had to fork it to get that NLTK install in there.

Would it be possible to split the `run-addon.yml` into something like `set-up-addon.yml` and `run-addon.yml`? Then, add-ons that need to install something before running could run those base add-ons.